openlibrary/macros/LoanStatus.html

```diff
@@ -43,6 +43,9 @@
       });
     </script>
 
+$elif ocaid and ctx.user and ctx.user.is_printdisabled():
```

Comment:
I think this should be after `is_readable`; they should see "Read" for public books that have no restrictions (for the same reason we don't show "borrow" for all books).